packages/vuepress/vuepress-theme-titanium/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vuepress-theme-titanium",
-  "version": "4.9.0",
+  "version": "4.9.1",
```

Comment:
You don't need to manually bump the version here, lerna will do it automatically for you by analyzing the commit messages when merging `develop` onto `master` ;)

Reply:
ah nice :+1: Hope this PR is not too bad :smile:

Reply:
Just a teeny tiny suggestion: You could use a a data attribute like `data-theme="dark|light"` instead of a class to make the toggle logic a little simpler. That helps you to avoid all the class contains/add/remove calls and you can simply set the value to the color mode. And in the css refer to is through `html[data-theme='dark']` to set the variables.

The rest looks perfectly fine to me!

Reply:
I like Jan's suggestion. Let's add that in a future release.